### DISCUSSION TITLE
Welcome to iOS-Open-GPX-Tracker Discussions!

### DISCUSSION THREAD

**Discussion by merlos** [merlos/iOS-Open-GPX-Tracker#223] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Frederik283** [comment]
Is there any way to specify the frequency of tracking, for example logging my position every x minutes? I use it for geotagging photos and the tagging app I use struggle with the large gpx files from a two hour session.

**Comment by merlos** [comment]
Right now the points are added when the location changes more than 2m. 

 I think there are apps that clean up the gpx files, but I can't recall any name...

**Comment by sgtarry** [comment]
I recently got Open GPX Tracker on my iPhone (v1.10.1), works great, except it always shows the time duration as 00:00. (In the upper right, where the sample screen shot in the App Store listing shows 00:37). How do I see the actual duration of a track?
